<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SWMM_results_dialog</class>
 <widget class="QDialog" name="SWMM_results_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SWMM_Results_Dialog</string>
  </property>
  <property name="accessibleName">
   <string>SWMM_Results_Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="layer_sel_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>SUBCATCHMENTS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>JUNCTIONS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OUTFALLS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DIVIDERS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>STORAGE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CONDUITS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PUMPS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ORIFICES</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WEIRS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OUTLETS</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="attr_select_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QgsFileWidget" name="out_file">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string>SWMM Out File</string>
   </property>
   <property name="accessibleDescription">
    <string>SWMM Out File</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="filter">
    <string>.out</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SWMM_results_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SWMM_results_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
